js/carousel.js

```diff
@@ -314,7 +314,18 @@
           e.preventDefault();
           e.stopPropagation();
         }
-        this._cycleTo(clickedIndex);
+
+        // fixes @materializecss/materialize#180
```

Comment:
Probably use the full GitHub URL as this is more helpful.

Reply:
Got it, I'll change that 👌

Reply:
renamed : https://github.com/materializecss/materialize/pull/187/commits/0ece6d8a03dfe92a14050ff4d9e2ac4f7c1d9719